roles/k8s_best_practices_certsuite/defaults/main.yml

```diff
@@ -14,6 +14,10 @@ kbpc_test_config:
 kbpc_accepted_kernel_taints: []
 kbpc_services_ignore_list: []
 kbpc_allow_preflight_insecure: false
+# CWE Hydra apikey for automated results submission
+# It should be created manually in Connect UI
+# Do not forget to define kbpc_cwe_project_id as well
+kbpc_cwe_apikey_path: "/opt/cache/cwe-apikey.txt"
```

Comment:
It would be better to not set a default for this since is required to submit, and document it in the README

Reply:
agree

Reply:
I'd prefer to keep the default value, the same way as we do for preflight, in /opt/cache/pyxis-apikey.txt
The partners are already used to it. I'll update the README